rdtools/test/test_normalize_with_expected_power.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+import pytest
+from rdtools.normalization import normalize_with_expected_power
+from pandas import Timestamp
+import numpy as np
+
+@pytest.fixture()
+def times_15():
+	return pd.date_range(start='20200101 12:00', end= '20200101 13:00', freq='15T')
+
+@pytest.fixture()
+def times_30():
+	return pd.date_range(start='20200101 12:00', end= '20200101 13:00', freq='30T')
+
+@pytest.fixture()
+def pv_15(times_15):
+	return pd.Series([1.0, 2.5, 3.0, 2.2, 2.1], index=times_15)
+
+@pytest.fixture()
+def expected_15(times_15):
+	return pd.Series([1.2, 2.3, 2.8, 2.1, 2.0], index=times_15)
+
+@pytest.fixture()
+def irradiance_15(times_15):
+	return pd.Series([1000.0, 850.0, 950.0, 975.0, 890.0], index=times_15)
+
+@pytest.fixture()
+def pv_30(times_30):
+	return pd.Series([1.0, 3.0, 2.1], index=times_30)
+
+@pytest.fixture()
+def expected_30(times_30):
+	return pd.Series([1.2, 2.8, 2.0], index=times_30)
+
+@pytest.fixture()
+def irradiance_30(times_30):
+	return pd.Series([1000.0, 950.0, 890.0], index=times_30)
+
+def test_normalize_with_expected_power_uniform_frequency(pv_15, expected_15, irradiance_15):
+	norm, insol = normalize_with_expected_power(pv_15, expected_15, irradiance_15)
+	expected_norm = pd.Series({Timestamp('2020-01-01 12:15:00', freq='15T'): 1.0,
+							   Timestamp('2020-01-01 12:30:00', freq='15T'): 1.0784313725490198,
```

Comment:
Indentation, also I have never seen this style of creating a Series before -- interesting

Reply:
Thanks for catching the formatting. Cleaning that up. I got that from `pd.Series.to_dict()` it seemed to be the most portable from the notebook I used to prototype the tests...